### PR TITLE
added: direct download button ✨

### PR DESCRIPTION
**Pull request by lisztomania23**

### CODE REVIEW

File: _layouts/default.html [hunk 1]
```diff
@@ -61,6 +61,10 @@ <h2 class="appPrice">
                             {% if site.appstore_link %}
                                 <a class="appStoreLink" href="{{site.appstore_link}}"><img class="appStore" src="/assets/appstore.png"></a>
                             {% endif %}
+                            {% if site.directDownload_link %}
+                        <a class="directDownloadLink" href="{{site.directDownload_link}}"><img class="directlink"
+                                src="/assets/download.png"></a>
+                        {% endif %}
```
Comment (TheLastProject) [inline _layouts/default.html:67]: Consistency:

```suggestion
                            {% if site.directDownload_link %}
                                <a class="directDownloadLink" href="{{site.directDownload_link}}"><img class="directlink" src="/assets/download.png"></a>
                        {% endif %}
```

**Review by TheLastProject — COMMENTED** [review]
Thanks! It looks pretty neat.

The "Catima is a community project" text sometimes touches the border of the background a bit much, but that's a minor thing.

I do worry a bit about how, on small screens like phones, the way the direct download button is next to the Google Play button it may confuse people in thinking it's Google Play. Maybe it should be a full-size button instead with the text "Direct Download" or something on it? That way nobody can ever get confused and it'll move to a new line on mobile automatically

image

**Comment by lisztomania23** [comment]
#7 instead of changing direct download button dialog warning on clicking it. tried changing size but it was breaking layout.
#33  positioned both download icon and contribute text for small screen

**Comment by TheLastProject** [comment]
I appreciate your work but I don't think it solves the issue, it is still super easy for people to get confused and think the download icon is for Google Play.

I played around with it a bit (to add a button for IzzyOnDroid) and it seems it should be much easier to add an extra button now at the end without breaking the layout since #36

**Pull request closed without merging**